### ISSUE TITLE
Add menu option to Show Queue

### ISSUE THREAD

**Issue by Wintandre** [FossifyOrg/Music-Player#33]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Music-Player/issues) or [closed](https://github.com/FossifyOrg/Music-Player/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Music-Player/discussions) or [closed](https://github.com/FossifyOrg/Music-Player/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Music-Player/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

Add a menu option 'Show Queue' (or similarly worded), which then starts the existing "Track queue" activitity.

### Why do you want this feature?

Currently to show the tracks in the queue, you have to press the icon on the track bar at the bottom, then press the icon again for the 'next' item. When using the app for the first time, this was not obvious and I actually thought that it wasn't possible to see the queue at all. Having it within the menu, would allow users to easily jump to the queue in order to manipulate it / clear it / etc.

**Comment by naveensingh** [comment]
@Wintandre 
Contribution guidelines will be organized in one place but check this please: https://github.com/FossifyOrg/General-Discussion?tab=readme-ov-file#contribution-rules-for-developers

**TL;DR:** Don't work on issues with the `needs triage` label. These issues are yet to be accessed and we might not want to implement everything that is suggested.

To me, a "Show queue" menu option doesn't seem like the best way to do it. Have you checked any other apps for inspiration?

**Comment by Wintandre** [comment]
Most recently I'd been using Auxio. There were others before that I can't remember. One did have the option in the menu though I accept that such a thing depends on the overall app UI style.

In Auxio, the queue is brought up in a similar way to Music-Player: tap on the track bar to bring up a full screen view of the playing track, then tap on the "Queue" label at the bottom of the screen. So the same number of taps required in the same place. However, with Auxio, due to the label, it's obvious how to get there.

Of course, being more familiar with Music-Player, it's now a simple thing to get to, but I thought it could be more helpful to new users by being more obvious.

An alternative could be to add an ImageButton within the TrackActivity's next track holder. There's a "queue music" vector available in Android Studio. Merely showing the image should be enough as the whole track holder layout has a click listener already on it to start the QueueActivity.

**Comment by Aga-C** [comment]
Better would be an icon, because "Queue" text could be problematic in different languages in terms of its length.

**Comment by Wintandre** [comment]
Yes, that's what I meant with "ImageButton", or strictly in this case it doesn't even need to be a button, so a normal ImageView would do it. For example..
![Music-Player](https://github.com/FossifyOrg/Music-Player/assets/84229910/0bd52f14-0630-4705-8ac6-934fe8ab7094)
